hints/aix.sh

```diff
@@ -694,5 +694,10 @@ EOF
   ;;
 esac # not gcc
 
+case "$osvers" in
```

Comment:
Commit message: "hints page"?  should this be "hints file"?

Reply:
yes, fixed